README.md

```diff
@@ -72,6 +72,10 @@ The web API is very simple. URLs are of the following format:
     `/etc/cvmfsmon/api.conf`.
 5.  \"geo\" - verifies that the geo api on a stratum 1 successfully
     responds with a server order for a test case on one repository.
+6.  \"whitelist\" - verifies that whitelist file is not expired. If
+    expiration time is less than 24 hours, a repository will be in
+    WARNING condition, and it will be in CRITICAL condition if whitelist
+    file is expired. The limit can be changed in `/etc/cvmfsmon/api.conf`.
```

Comment:
Change "verifies that whitelist" to "verifies that the .cvmfswhitelist", and later "if whitelist" to "if the whitelist".  Also change "24 hours" to "24 hours away (by default)".

Reply:
On second thought, let's make the default be 48 hours instead of 24.  Here and of course also in api.conf and cvmfsmon_api.py.